src/components/SearchBar.css

```
@@ -14,6 +14,6 @@ label {
     font-family: Arial, sans-serif;
     font-weight: bold;
     font-size: 20px;
-    color: #16552d;
+    color: crimson;
```

```suggestion
    color: blue;
```
remember that the new agreement is blue